structured_external_storage_all/models/ir_attachment_sync_rule.py

```diff
@@ -30,9 +30,9 @@ def _get_model_selection(self):
     location_id = fields.Many2one(
         'external.file.location', 'Sync Name', required=True)
     domain = fields.Char(
-        'Domain', default=[(1,'=',1)],
-        help='Conditions that will apply to the '
-        'select model. E.g [(amount, '>', 500, )]')
+        'Domain', default=[],
+        help="Conditions that will apply to the "
+        "select model. E.g [(amount, ">", 500, )]")
```

Comment:
This is still not good. The string effectively ends at the ", then comes the `>`, and then the string restarts again. You have to escape the `"` by prepending a backslash: `\"`, or you have to use single quotes.